### ISSUE TITLE
about pretrained models

### ISSUE THREAD

**Issue by fire717** [epic-kitchens/epic-sounds-annotations#11]
Hi, thx for your baseline models.
I see two models here: SLOWFAST_EPIC_SOUNDS.pyth and SLOWFAST_VGG.pyth, and a little confused, 
is this right?

1. SLOWFAST_VGG.pyth is the base pretained model which trained on Other datasets like Imagenet, and load SLOWFAST_VGG.pyth to train can get SLOWFAST_EPIC_SOUNDS.pyth?
2. In order to paticipate the competition, do I need to train from SLOWFAST_VGG.pyth , or I can load SLOWFAST_EPIC_SOUNDS.pyth and only train a few epoch, as long as I can get better score, then I can commit it to the leaderboard?

**Comment by JacobChalk** [comment]
Hi,

1. Correct, SLOWFAST_VGG.pyth is the checkpoint file for the SlowFast model pretrained on the VGGSound dataset. SLOWFAST_VGG.pyth was used in order to fine-tune our model on EPIC-Sounds, obtaining the SLOWFAST_EPIC_SOUNDS.pyth checkpoint file. Indeed, if you load SLOWFAST_VGG.pyth with the same config we used for training, you should be able to replicate our results (or at least get close to them, accounting for randomness in training). 
2. You may use whatever method you please in order to participate in the competition, whether that is starting from the VGG weights, or EPIC-SOUNDS weights. Please note that the challenge does come with a technical report of 2-6 pages detailing your method, so your method should allow for sufficient explanation as to why your scores are better

**Comment by fire717** [comment]
Got it, and one last question, does the method need to be innovative? Such as just change the backbone (use new pretrained model on imagenet) or add one simple data augmentation.
Still say we need to do some innovative methods such new model structure or new fancy loss function, etc.

**Comment by dimadamen** [comment]
No novelty requirement is expected for participating or winning any challenges. You could simply replace the backbone if you wish. The report has though to be very clear about what was done, so others could replicate the effort.
